.github/workflows/CI-tag-orchestrator.yml

```diff
@@ -5,103 +5,87 @@ on:
     tags:
       - "[0-9]+.[0-9]+.[0-9]+-[0-9]+.[0-9]+.[0-9]+*"
   workflow_dispatch:
+    inputs:
+      git_tag:
+        type: string
+    secrets:
+      DOCKER_HUB_USERNAME:
+        required: true
+      DOCKER_HUB_TOKEN:
+        required: true
 
 env:
   RUST_BACKTRACE: 1
 
 jobs:
-  execute-system-tests:
-    name: Execute System Tests
-    uses: HorizenLabs/zkVerify-qa/.github/workflows/CI-e2e-tests.yml@main
+  env-setup:
+    uses: ./.github/workflows/CI-build.yml
+    with:
+      CACHING_MODE: "CACHE_SAVE"
```

Comment:
It seems to me that the build files should be generated again for this workflow, thus setting it to "CACHE_SAVE"

Or rather, it seems that these folders can be used from the cache
```            
~/.cargo/bin/
~/.cargo/registry/index/
~/.cargo/registry/cache/
~/.cargo/git/db/
```
But the `/target` folder should be regenerated.

Lmk if that is correct @DanieleDiBenedetto @lander86

Reply:
@la10736

Reply:
Anyway I noted that caching mode is required (I cannot understand why... maybe is a wrong requirement)... so you can use `""` 

We should add to backlog if is the case to remove this requirement